kiva/fonttools/font_manager.py

```diff
@@ -47,1000 +29,58 @@
             see license/LICENSE_TTFQUERY.
 """
 import errno
-import glob
 import logging
 import os
 import pickle
-import subprocess
-import sys
 import tempfile
 import warnings
 
-from fontTools.ttLib import TTCollection, TTFont, TTLibError
-
 from traits.etsconfig.api import ETSConfig
 
-from . import afm
-from ._score import (
+from kiva.fonttools._scan_parse import create_font_list
```

Comment:
nit: these come from underscore-prefixed modules, but themselves don't contain an underscore. Probably overkill and not necessary to change, just an observation

Reply:
Those are the "protected" functions, to borrow from C++... Basically, the underscore-prefixed functions in those underscore-prefixed modules should never be imported by anything save for unit tests. The functions that we do import are safe to use in other modules, but not to be used by external code.